web/packages/teleport/src/Discover/Shared/Mark.ts

```diff
@@ -18,7 +18,7 @@
 
 import styled from 'styled-components';
 
-export const Mark = styled.mark`
+export const Mark = styled.mark<{ light?: boolean }>`
```

Comment:
`light` doesn't seem to be used anywhere. But I guess we can keep it if we just want to avoid breaking anything.